### PR TITLE
feat: Update ACI tooltip copy

### PR DESCRIPTION
**Pull request by pstakoun**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by mike-plummer — COMMENTED** [review]
Changes look okay, but you've got some e2e test failures. See `settings.cy.ts`

**Review by mike-plummer — APPROVED** [review]
👍 Only comment is the discrepancy between sometimes saying "the Dashboard" and other times "the Cypress Dashboard". I saw y'all were discussing that earlier so I'll assume everyone is on board with it as-is

File: packages/frontend-shared/src/locales/en-US.json [hunk 1]
```diff
@@ -426,7 +426,7 @@
       "errorNotFound": "Project not found",
       "errorNotFoundButton": "Connect it again",
       "errorNotLoggedIn": "You are not connected to the Cypress Dashboard",
-      "errorNotLoggedInButton": "Log in to Cypress Dashboard"
+      "errorNotLoggedInButton": "Log in to the Cypress Dashboard"
```
Comment (pstakoun) [inline packages/frontend-shared/src/locales/en-US.json:429]: I'm trying to figure out where this is used - can't find usages of `errorNotLoggedInButton`.

Comment (mike-plummer) [reply packages/frontend-shared/src/locales/en-US.json:429]: Looks like several of these strings aren't referenced (most of the `error*` entries under `recordKey` anyways)

**Pull request merged**